src/main/java/seedu/address/model/recipe/InstructionList.java

```diff
@@ -1,41 +1,46 @@
 package seedu.address.model.recipe;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.commons.util.AppUtil.checkArgument;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
+
+import seedu.address.logic.parser.ItemListParser;
+import seedu.address.logic.parser.exceptions.ParseException;
 
 /**
  * Represents a Recipe's instructions in the recipe book.
  */
 public class InstructionList {
 
-    public static final String MESSAGE_CONSTRAINTS =
-            "Instructions can take any values for now, and it should not be " + "blank";
+    public static final String MESSAGE_CONSTRAINTS = "Instructions can take any values for now, and it should not be "
+            + "blank";
 
     /*
-     * The first character of the instructions must not be a whitespace,
-     * otherwise " " (a blank string) becomes a valid input.
+     * The first character of the instructions must not be a whitespace, otherwise
+     * " " (a blank string) becomes a valid input.
      */
     public static final String VALIDATION_REGEX = "[^\\s].*";
 
-    public final List<Instruction> instructions;
-
-    // TODO: change to List, for now using String to temp store instruction list
-    public final String instructionListString;
+    public final List<Instruction> data;
```

Comment:
I think data needs to be renamed to something more specific, as per the requirements for the coding standards.

Perhaps `instructionData`?